### PR TITLE
revert to use old client

### PR DESCRIPTION
**Pull request by wbgilmartin**
changes in install.sh to default to the old tensile client

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -10,25 +10,25 @@ function display_help()
...
-      -o | --cov                Set Tensile code_object_version (V2 or V3)
-      -t | --test_local_path    Use a local path for Tensile instead of remote GIT repo
-           --cpu_ref_lib        Specify library to use for CPU reference code in testing (blis or lapack)
-           --hip-clang          Build library for amdgpu backend using hip-clang
-           --build_dir          Specify name of output directory (default is ./build)
-      -n | --no_tensile         Build subset of library that does not require Tensile
-      -r | --no-tensile-host    Do not build with Tensile host
-      -u | --use-custom-version Use user-specified Tensile version
-           --skipldconf         Skip ld.so.conf entry
-           --ignore-cuda        Ignores installed cuda version and builds with rocm stack instead
```
Comment (leekillough) [inline install.sh]: We don't want to remove `--use-custom-version` or `--ignore-cuda`, do we?

Comment (amdkila) [reply install.sh]: That's correct. Must've been removed when clearing up a nearby merge conflict.

Comment (leekillough) [reply install.sh]: I think `--use-custom-version` was renamed `--use-tag-only`, but it needs to be replaced in all places.

**Comment by leekillough** [comment]
Here's my local changes to this PR:
```
diff --git a/install.sh b/install.sh
index 35317e8e..f0989ebe 100755
--- a/install.sh
+++ b/install.sh
@@ -26,6 +26,7 @@ rocBLAS build & installation helper script
            --build_dir         Specify name of output directory (default is ./build)
       -n | --no_tensile        Build subset of library that does not require Tensile
       -u | --use-tag-only      Ignore Tensile version and just use the Tensile tag
+           --ignore-cuda       Ignores installed cuda version and builds with rocm stack instead
            --skipldconf        Skip ld.so.conf entry
 EOF
 #          --prefix             Specify an alternate CMAKE_INSTALL_PREFIX for cmake
@@ -267,7 +268,7 @@ fi
 # check if we have a modern version of getopt that can handle whitespace and long parameters
 getopt -T
 if [[ $? -eq 4 ]]; then
-  GETOPT_PARSE=$(getopt --name "${0}" --longoptions help,install,clients,dependencies,debug,hip-clang,no_tensile,logic:,architecture:,cov:,fork:,branch:,build_dir:,test_local_path:,cpu_ref_lib:,use-custom-version:,skipldconf,ignore-cuda --options nhicdgl:a:o:f:b:t:u: -- "$@")
+  GETOPT_PARSE=$(getopt --name "${0}" --longoptions help,install,clients,dependencies,debug,hip-clang,no_tensile,logic:,architecture:,cov:,fork:,branch:,build_dir:,test_local_path:,cpu_ref_lib:,use-tag-only:,skipldconf,ignore-cuda --options nhicdgl:a:o:f:b:t:u: -- "$@")
 else
   echo "Need a new version of getopt"
   exit 1
@@ -335,7 +336,7 @@ while true; do
     --skipldconf)
         skip_ld_conf_entry=true
         shift ;;
-    -u|--use-custom-version)
+    -u|--use-tag-only)
         tensile_version=${2}
         shift 2;;
     --prefix)
```

**Comment by amdkila** [comment]
> Here's my local changes to this PR:
> 
> ```
> diff --git a/install.sh b/install.sh
> index 35317e8e..f0989ebe 100755
> --- a/install.sh
> +++ b/install.sh
> @@ -26,6 +26,7 @@ rocBLAS build & installation helper script
>             --build_dir         Specify name of output directory (default is ./build)
>        -n | --no_tensile        Build subset of library that does not require Tensile
>        -u | --use-tag-only      Ignore Tensile version and just use the Tensile tag
> +           --ignore-cuda       Ignores installed cuda version and builds with rocm stack instead
>             --skipldconf        Skip ld.so.conf entry
>  EOF
>  #          --prefix             Specify an alternate CMAKE_INSTALL_PREFIX for cmake
> @@ -267,7 +268,7 @@ fi
>  # check if we have a modern version of getopt that can handle whitespace and long parameters
>  getopt -T
>  if [[ $? -eq 4 ]]; then
> -  GETOPT_PARSE=$(getopt --name "${0}" --longoptions help,install,clients,dependencies,debug,hip-clang,no_tensile,logic:,architecture:,cov:,fork:,branch:,build_dir:,test_local_path:,cpu_ref_lib:,use-custom-version:,skipldconf,ignore-cuda --options nhicdgl:a:o:f:b:t:u: -- "$@")
> +  GETOPT_PARSE=$(getopt --name "${0}" --longoptions help,install,clients,dependencies,debug,hip-clang,no_tensile,logic:,architecture:,cov:,fork:,branch:,build_dir:,test_local_path:,cpu_ref_lib:,use-tag-only:,skipldconf,ignore-cuda --options nhicdgl:a:o:f:b:t:u: -- "$@")
>  else
>    echo "Need a new version of getopt"
>    exit 1
> @@ -335,7 +336,7 @@ while true; do
>      --skipldconf)
>          skip_ld_conf_entry=true
>          shift ;;
> -    -u|--use-custom-version)
> +    -u|--use-tag-only)
>          tensile_version=${2}
>          shift 2;;
>      --prefix)
> ```

Looks good for the most part. However, I renamed the flag from use-tag-only to use-custom-version. The idea is to use the old Tensile version if the Tensile tag was prematurely updated. So it isn't really about the tag, so I thought my initial name for the flag was misleading.

File: install.sh [hunk 2]
```diff
@@ -10,27 +10,26 @@ function display_help()
...
+      -b | --branch              GitHub branch or tag to use, e.g., develop, mybranch or <commit hash>
+      -l | --logic               Set Tensile logic target, e.g., asm_full, asm_lite, etc.
+      -a | --architecture        Set Tensile GPU architecture target, e.g. all, gfx000, gfx803, gfx900, gfx906, gfx908
+      -o | --cov                 Set Tensile code_object_version (V2 or V3)
+      -t | --test_local_path     Use a local path for Tensile instead of remote GIT repo
+           --cpu_ref_lib         Specify library to use for CPU reference code in testing (blis or lapack)
+           --hip-clang           Build library for amdgpu backend using hip-clang
+           --build_dir           Specify name of output directory (default is ./build)
+      -n | --no_tensile          Build subset of library that does not require Tensile
+      -u | --use-custom-version  Ignore Tensile version and just use the Tensile tag
```
Comment (leekillough) [inline install.sh]: Sorry, I seem to have removed `--tensile-host` and `--no-tensile-host` from the help message. We should add them back.

Comment (amcamd) [reply install.sh]: commit below adds them back

**Comment by leekillough** [comment]
https://github.com/wbgilmartin/rocBLAS/pull/10

**Comment by amdkila** [comment]
> Can we please resolve the conflicts so that CI can run its course? Thanks.

Fixed the tensile_tag conflict

**Pull request merged**